lib/Sema/DeclSpec.cpp

```diff
@@ -975,7 +975,7 @@ bool DeclSpec::setFunctionSpecUnchecked(SourceLocation Loc,
 bool DeclSpec::setFunctionSpecForany(SourceLocation Loc,
                                         const char *&PrevSpec,
                                         unsigned &DiagID) {
-  if (FS_forany_specified) {
+  if (FS_itypeforany_specified || FS_forany_specified) {
```

Comment:
Is the case for itypeforany_specified a conflicting declspec, not a duplicate declspec?